### PR TITLE
MAR-4055-supplier-frontend-change-brief-to-opportunity

### PR DESCRIPTION
**Pull request by Revit**

### CODE REVIEW

**Review by Reshma-M-A — COMMENTED** [review]
Found instances of brief in the file:
app/main/helpers/briefs.py

File: app/main/helpers/briefs.py [hunk 1]
```diff
@@ -160,7 +160,8 @@ def send_brief_clarification_question(data_api_client, brief, clarification_ques
         )
     except EmailError as e:
         current_app.logger.error(
-            "Brief question email failed to send. error={error} supplier_code={supplier_code} brief_id={brief_id}",
+            "Opportinity question email failed to send. error={error}"
```
Comment (Reshma-M-A) [inline app/main/helpers/briefs.py:163]:
```suggestion
            "Opportunity question email failed to send. error={error}"
```

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,8387 @@
+{
```
Comment (Reshma-M-A) [inline package-lock.json:1]: We can remove this file.

File: tests/app/main/test_suppliers.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 from lxml import html
 import pytest
 
-
+"""d"""
```
Comment (Reshma-M-A) [inline tests/app/main/test_suppliers.py:13]: Is there a reason we need this here? If not, perhaps we can remove it.

File: app/main/helpers/briefs.py [hunk 4]
```diff
@@ -160,7 +160,8 @@ def send_brief_clarification_question(data_api_client, brief, clarification_ques
         )
     except EmailError as e:
         current_app.logger.error(
-            "Brief question email failed to send. error={error} supplier_code={supplier_code} brief_id={brief_id}",
+            "Opportunity question email failed to send. error={error}"
+            " supplier_code={supplier_code} brief_id={brief_id}",
```
[the remark this answers is not in the document]
Comment (Revit) [reply app/main/helpers/briefs.py:164]: changed as suggested

**Review by tobiasbrown — APPROVED** [review]

**Pull request merged**